scripts/deps-ppa/static_z3.sh

```diff
@@ -179,7 +198,7 @@ This program is free software: you can redistribute it and/or modify
  Public License version 3 can be found in "/usr/share/common-licenses/GPL-3".
 EOF
 cat <<EOF > debian/changelog
-libz3-static-dev (0.0.1-1ubuntu0) saucy; urgency=low
+z3-static (0.0.1-1ubuntu0) saucy; urgency=low
```

Comment:
Why `0.0.1` and not `4.8.12`?

Reply:
No other reason than that the script is largely a copy of scripts/release_ppa.sh, which also does it in this weird manner :-).